app/helpers/account_helper.rb

```diff
@@ -19,4 +19,10 @@ def flash_as_notice(notice)
       I18n.t("devise.registrations.update_needs_confirmation"),
     ].include? notice
   end
+
+  def confirmation_banner_prompt_type
+    return "updating" if confirmed_user_changed_email?
+
+    "setting up"
```

Comment:
I think I'd feel slightly better with this text in the locale file.  Maybe jut two variants of the `intro` entry?

Reply:
can do!

Reply:
Done here: https://github.com/alphagov/govuk-account-manager-prototype/pull/560/commits/08657f358e6b214ddb39e20b88d9fc008cc5d2de